iceoryx_examples/ice_multi_publisher/README.md

```diff
@@ -1,5 +1,7 @@
 # Multiple publishers for the same topic
 
+<!-- Rewrite this document to re-use icedelivery -->
```

Comment:
I wouldn't rewrite this document until we have decided if we will keep this example.

Reply:
As this example doesn't teach users anything in advance I'd suggest to remove it before v1.0 Adapted comment

Reply:
Removed the comment